app/rest/runtime/pom.xml

```diff
@@ -972,12 +972,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
-      <groupId>com.h2database</groupId>
-      <artifactId>h2</artifactId>
-      <scope>test</scope>
```

Comment:
Shouldn't we keep this one and remove the `<scope>runtime</scope>` one, we're not depending on h2 at runtime right?

Reply:
@zregvart y that seems reasonable to me.